### PR TITLE
Disable additive animations for backgroundColor and opacity.

### PR DESCRIPTION
**Pull request by jverkoey**
Closes https://github.com/material-motion/motion-animator-objc/issues/61

### CODE REVIEW

**Review by samnm — DISMISSED** [review]

**Comment by jverkoey** [comment]
This is blocked due to some incorrect behavior with non-additive animations. Will be sending out PRs to resolve that behavior first before proceeding with this.

**Comment by jverkoey** [comment]
The necessary fixes have landed and this PR is ready for review.

**Comment by jverkoey** [comment]
Gentle ping.

File: src/private/CABasicAnimation+MotionAnimator.m [hunk 1]
```diff
@@ -44,6 +44,16 @@ static BOOL IsCGSizeType(id someValue) {
   return NO;
 }
 
+static BOOL IsInAdditiveAnimationBlacklist(NSString *keyPath) {
```
Comment (samnm) [inline src/private/CABasicAnimation+MotionAnimator.m]: IMO isAnimationKeyAlwaysNonadditive would be a better name. `Blacklist` is a little unintuitive.

Comment (samnm) [inline src/private/CABasicAnimation+MotionAnimator.m]: IMO something like `isAnimationKeyAlwaysNonadditive` would be a better name.

**Review by samnm — DISMISSED** [review]

File: src/private/CABasicAnimation+MotionAnimator.m (hunk 1 above)
Comment (jverkoey) [reply src/private/CABasicAnimation+MotionAnimator.m]: Done.

**Review by samnm — APPROVED** [review]

**Pull request merged**